packages/scripts.nix

```diff
@@ -16,6 +16,82 @@
     text = builtins.readFile ./destroy-coco-aks.sh;
   };
 
+  upload-image = writeShellApplication {
+    name = "upload-image";
+    runtimeInputs = with pkgs; [
+      azure-cli
+      gnused
+      uplosi
+    ];
+    text =
+      let
+        image = pkgs.image-podvm;
+      in
+      ''
+        subscriptionId=""
+        location="GermanyWestCentral"
+        resourceGroup=""
+
+        for i in "$@"; do
+          case $i in
+          --subscription-id=*)
```

Comment:
You could do `az account show -o tsv --query id`, which would avoid the hard-coded Edgeless subscription.

Reply:
This doesn't work if the user is part of multiple subscriptions though, does it?

Reply:
It would show the default subscription. Maybe we should default to`az account show` but allow overriding with an explicit flag for the rare use case?

Reply:
We don't really hard-code this anyway, the value can be changed in `justfile.env`. We only hard-code the default value. Interpolating values into the just variable seems a bit painful, so let's just keep hard-coding the Edgeless subscription.